subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+
+test_id = -1
+
+for x in range(0, len(name_of_tests) - 1):
+    if test_request == name_of_tests[x]:
+        test_id = x
+
+print('test_id=' + str(test_id))
+
+cap_pcap_file = str(arguments[2])
+#capture.pcap
+
+device_address = '127.0.0.1'
+
+if test_id == 0:
+    device_address = str(arguments[3])
+
+report_filename = 'report.txt'
+
+packets_in_report = 10
+
+tcpdump_display_all_packets = 'tcpdump -n src host ' + device_address + ' -r ' + cap_pcap_file
+tcpdump_display_udp_bacnet_packets = 'tcpdump -n udp dst portrange 47808-47809 ' + cap_pcap_file
+tcpdump_display_arp_packets = 'tcpdump -v arp -r ' + cap_pcap_file
+tcpdump_display_ntp_packets = 'tcpdump dst port 123 -r ' + cap_pcap_file
+
+tests = [tcpdump_display_all_packets, tcpdump_display_udp_bacnet_packets, tcpdump_display_arp_packets, tcpdump_display_ntp_packets]
```

Comment:
why don't you use the same list as elsewhere?

I think this might be cleaner if you had them both in the same python file, because then you could just have one list of tests and not muck about with all the indirection.

Reply:
fixed